helm/prometheus-rules/templates/platform/atlas/alerting-rules/promtail.rules.yml

```diff
@@ -52,3 +52,27 @@ spec:
             cancel_if_cluster_status_creating: "true"
             cancel_if_cluster_status_deleting: "true"
             cancel_if_cluster_status_updating: "true"
+        - alert: PromtailConflictsWithAlloy
+          # This alert was created as a workaround of https://github.com/giantswarm/giantswarm/issues/33341
```

Comment:
Can you had a TODO: remove when all clusters are >= v29 (I think)

Reply:
Well, we'll remove the whole promtail rules file then I guess.

Reply:
Sure